### PR TITLE
[Localization] #1761 Korean trainer dialogue (youngster, lass)

### PR DESCRIPTION
**Pull request by Ohmry**
## What are the changes?

Translating locales\ko\dialogue.ts into Korean.
This PR is added korean dialogue for Gym leader in Kanto region

@sodaMelon @Ohmry @returntoice now participating in https://github.com/pagefaultgames/pokerogue/issues/1761 translating project.

## Why am I doing these changes?

Few trainers's dialogue have been translated into Korean yet.

## What did change?

Translated dialogue of youngster, lass to Korean.

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [ ] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by Ohmry** [comment]
@sodaMelon 말씀해주신대로 전각문자 1개로 되어있는 말줄임표로 변경하였습니다. 시간되실 때 확인 부탁드립니다!

**Review by sodaMelon — APPROVED** [review]
LGTM. 👍 Thank you for understanding the letter and half letter. (on pokemon font)

File: src/locales/ko/dialogue.ts [hunk 1]
```diff
@@ -4,58 +4,58 @@ import { DialogueTranslationEntries, SimpleTranslationEntries } from "#app/plugi
...
+      4: "재밌는 포켓몬 배틀하자!",
+      5: "내가 포켓몬을 어떻게 다뤄야하는지 보여줄게!",
+      6: "진정한 배틀은 진지한 자세부터 시작이야! 준비됐어?",
+      7: "젊음이 한순간이듯 배틀에서 네 기회도 단 한번만 주어질거야. 곧 넌 추억속으로 사라질거야.",
+      8: "나에겐 살살해도 돼, 알았지? 그래도 난 진지하게 싸울거야!",
+      9: "학교는 지겨워. 나는 할 일이 없어. 하암~ 난 그저 시간을 때우기 위해 싸울뿐이야."
+    },
+    "victory": {
+      1: "인상적이었어! 난 아직 배울게 많구나.",
+      2: "네가 날 이렇게 심하게 때릴 줄이야…",
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:51]:
```suggestion
      2: "내가 이렇게까지 크게 질 줄은 몰랐어…",
```
여기서 beat은 이긴다는 의미로 봐야 할 듯합니다. 때린다고 하면 마치 주인공이 폭행범(?)이 되어 버린 것 같아지네요ㅋㅋ

**Review by returntoice — CHANGES_REQUESTED** [review]
몇 가지 내용이나 맞춤법 수정을 했습니다.

**Comment by Ohmry** [comment]
@returntoice  😀 감사합니다.

**Review by returntoice — APPROVED** [review]

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**